tools/db_crashtest.py

```diff
@@ -802,8 +782,9 @@ def finalize_and_sanitize(src_params):
     # Remove the following once write-prepared/write-unprepared with/without
     # unordered write supports timestamped snapshots
     if dest_params.get("create_timestamped_snapshot_one_in", 0) > 0:
-        dest_params["txn_write_policy"] = 0
         dest_params["unordered_write"] = 0
+        if dest_params.get("txn_write_policy", 0) != 0:
```

Comment:
Do we still get to test "txn_write_policy": random.randint(0, 2) combined with "unordered_write": random.randint(0, 1) then with this nested if and initial `create_timestamped_snapshot_one_in  > 0`?

Reply:
Line 786 and 787 here are just disabling create_timestamped_snapshot_one_in for WP/WUP as was done before by `multiops_wp_txn_params` and the `dest_params["txn_write_policy"] = 0` line. I'm not changing the conditions for `unordered_write` best I can tell, and avoiding having db_crashtest.py MODIFY txn_write_policy since we want to pass that in authoritatively.

Reply:
> and initial create_timestamped_snapshot_one_in > 0?

Ohh right now that when I looked again this random combination wasn't tested when create_timestamped_snapshot_one_in > 0